src/status_im/ui/components/icons/vector_icons.cljs

```diff
@@ -2,121 +2,198 @@
-            :icons/chats               (components.svg/slurp-svg "./resources/icons/chats.svg")
-            :icons/hamburger           (components.svg/slurp-svg "./resources/icons/hamburger.svg")
-            :icons/hidden              (components.svg/slurp-svg "./resources/icons/hidden.svg")
-            :icons/in-contacts         (components.svg/slurp-svg "./resources/icons/in_contacts.svg")
-            :icons/lock                (components.svg/slurp-svg "./resources/icons/lock.svg")
-            :icons/mic                 (components.svg/slurp-svg "./resources/icons/mic.svg")
-            :icons/ok                  (components.svg/slurp-svg "./resources/icons/ok.svg")
-            :icons/public              (components.svg/slurp-svg "./resources/icons/public.svg")
-            :icons/public-chat         (components.svg/slurp-svg "./resources/icons/public_chat.svg")
-            :icons/qr                  (components.svg/slurp-svg "./resources/icons/QR.svg")
-            :icons/input-commands      (components.svg/slurp-svg "./resources/icons/input_commands.svg")
-            :icons/input-send          (components.svg/slurp-svg "./resources/icons/input_send.svg")
-            :icons/back                (components.svg/slurp-svg "./resources/icons/back.svg")
-            :icons/forward             (components.svg/slurp-svg "./resources/icons/forward.svg")
-            :icons/dropdown-up         (components.svg/slurp-svg "./resources/icons/dropdown_up.svg")
-            :icons/up                  (components.svg/slurp-svg "./resources/icons/up.svg")
-            :icons/down                (components.svg/slurp-svg "./resources/icons/down.svg")
-            :icons/grab                (components.svg/slurp-svg "./resources/icons/grab.svg")
-            :icons/share               (components.svg/slurp-svg "./resources/icons/share.svg")
-            :icons/tooltip-triangle    (components.svg/slurp-svg "./resources/icons/tooltip-triangle.svg")
-            :icons/open                (components.svg/slurp-svg "./resources/icons/open.svg")
-            :icons/network             (components.svg/slurp-svg "./resources/icons/network.svg")
-            :icons/wnode               (components.svg/slurp-svg "./resources/icons/wnode.svg")
-            :icons/refresh             (components.svg/slurp-svg "./resources/icons/refresh.svg")
-            :icons/newchat             (components.svg/slurp-svg "./resources/icons/newchat.svg")
-            :icons/logo                (components.svg/slurp-svg "./resources/icons/logo.svg")
-            :icons/camera              (components.svg/slurp-svg "./resources/icons/camera.svg")
-            :icons/check               (components.svg/slurp-svg "./resources/icons/check.svg")
-            :icons/dots                (components.svg/slurp-svg "./resources/icons/dots.svg")
-            :icons/warning             (components.svg/slurp-svg "./resources/icons/warning.svg")})
+(if platform/desktop?
```

Comment:
wouldn't this work and avoid duplication ?

`(def slurp-svg (if plateform/desktop? js/require components.svg/slurp-svg))`
then use slurp-svg